notebooks/_lesson02_data-preparation.ipynb

```diff
@@ -66,8 +66,10 @@
      "name": "stdout",
```

Comment:
<p>just cosmetics, but maybe you want to get rid of this?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lewtun/dslectures/pull/15/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>done! we should also find a way to not download files if they already exist</p>